pkg/operator/apis/monitoring/v1/rules_types.go

```diff
@@ -192,5 +204,5 @@ type Rule struct {
 
 // RulesStatus contains status information for a Rules resource.
 type RulesStatus struct {
-	// TODO: add status information.
+	MonitoringStatus `json:",inline"`
```

Comment:
I don't see in the CRD. Do you need to run `make regen`?

Reply:
It's in there (the `properties` field). Correct me if I'm wrong.

Reply:
I see it. Thanks!